pydantic_ai_slim/pydantic_ai/mcp.py

```diff
@@ -64,10 +64,12 @@ class MCPServer(AbstractToolset[Any], ABC):
     read_timeout: float = 5 * 60
     process_tool_call: ProcessToolCallback | None = None
     allow_sampling: bool = True
-    max_retries: int = 1
     sampling_model: models.Model | None = None
+    max_retries: int = 1
     # } end of "abstract fields"
 
+    _id: str | None = field(init=False, default=None)
```

Comment:
```suggestion
    _id: str | None
```
since we have an explicit `__init__` the `= field(...)` is not necessary any more, right?